evalml/tests/automl_tests/test_automl_search_classification.py

```diff
@@ -1597,17 +1206,19 @@ def test_time_series_pipeline_parameter_warnings(
             X_train=X,
             y_train=y,
             problem_type="time series binary",
-            max_batches=2,
+            max_batches=1,
```

Comment:
The story of this test is that it checks for `Logistic Regression` but this is not used in `Default`. Candidate for mock algorithm.